### PR TITLE
OW-744: Holo app unity integration refactor

### PR DESCRIPTION
**Pull request by joliveraortega**

### CODE REVIEW

File: Unity/opentok/holo/holo/ViewController.mm [hunk 1]
```diff
@@ -41,46 +46,135 @@
 static double widgetHeight = 240;
 static double widgetWidth = 320;
 
+static Boolean const kUnityRenderingEnabled = YES;
```
Comment (gmininberg) [inline Unity/opentok/holo/holo/ViewController.mm:49]: BOOL

File: Unity/opentok/holo/holo/ViewController.mm [hunk 2]
```diff
@@ -41,46 +46,135 @@
...
 @property (nonatomic) OTSubscriber *subscriber;
 @property (nonatomic) UILabel* publisherStatsLabel;
 @property (nonatomic) UILabel* subscriberStatsLabel;
 @end
 
 @implementation ViewController {
     UIView *_view;
+#if !(TARGET_IPHONE_SIMULATOR)
+#ifndef SKIP_USING_VONAGE_EHC_SUBSCRIBER_RENDERER
+    UIView *_unityView;
```
Comment (gmininberg) [inline Unity/opentok/holo/holo/ViewController.mm:93]: better to be a `@property`

**Review by gmininberg — COMMENTED** [review]
i miss this line in your code:
```
[self presentViewController:[[[self unityFramework] appController] rootViewController] animated:YES completion:nil];
```
with my testing adding the subview didn't work since `rootViewController` controls the lifecycle of GL work.

**Comment by joliveraortega** [comment]
> i miss this line in your code:
> 
> ```
> [self presentViewController:[[[self unityFramework] appController] rootViewController] animated:YES completion:nil];
> ```

I had it but finally removed it. I understood it was needed to add the local view. Holo app follows either a sender or receiver approach and I considered there is no Unity integration for the sender so I could just add the local view without any Unity involvement.

> with my testing adding the subview didn't work since `rootViewController` controls the lifecycle of GL work.

File: Unity/opentok/holo/holo/ViewController.mm (hunk 2 above)
Comment (joliveraortega) [reply Unity/opentok/holo/holo/ViewController.mm]: Done

File: Unity/opentok/holo/holo/ViewController.mm (hunk 1 above)
Comment (joliveraortega) [reply Unity/opentok/holo/holo/ViewController.mm]: Done

**Review by gmininberg — APPROVED** [review]

**Pull request merged**